api/ApiHandler.py

```diff
@@ -1,21 +1,34 @@
-from flask_restful import Api, Resource, request
-
-class ApiHandler(Resource):
-  def get(self):
-    search_term = request.args.get("query")
-    ## TODO: return err if query is invalid twitter handle
-    ## TODO: use search_term to return proper array of words
-    return {
-      'resultStatus': 'SUCCESS',
-      'words': [
-                {
-                    "text": 'told',
-                    "value": 64
-                },
-                {
-                    "text": "hi", 
-                    "value": 1
-                }
-            ]
-      }
+from flask_restful import Resource
+import requests;
+import os;
 
+from flask import Flask, jsonify, json
+
+
+class ApiHandelr(Resource):
+  url = 'https://api.twitter.com/1.1/statuses/user_timeline.json'
+  Twitter_Barear_Token= os.environ.get('Twitter_Barear_Token')
+
+  header = {"Authorization":  f"Bearer {Twitter_Barear_Token}"}
+  def getTweets(self):
```

Comment:
you need to keep this as `def get(self):`

this is because in flask_restful def get maps directly to the get request. I think that's why you were seeing the error for get not found :)

Reply:
done!